components/content-service/pkg/service/usagereport-service.go

```diff
@@ -18,25 +18,22 @@ import (
 	"github.com/gitpod-io/gitpod/content-service/pkg/storage"
 )
 
-const (
-	usageReportBucketName = "usage-reports"
-)
-
 // UsageReportService implements UsageReportServiceServer
 type UsageReportService struct {
-	cfg config.StorageConfig
-	s   storage.PresignedAccess
+	cfg        config.StorageConfig
+	s          storage.PresignedAccess
+	bucketName string
```

Comment:
Question: any reason that we did not put the bucketName variable under the StorageConfig data struct?

Reply:
It could probably go there but it doesn't really fit as it's not related to credentials or accessing the storage in general; it's only relevant for the usage report service.